content/articles/okta-identity-provider.markdown

```diff
@@ -42,27 +42,48 @@ Once they are granted access, they can log in directly via Okta. If the user log
 
 Follow the instructions below to connect DNSimple to your Okta organization.
 
-### Adding the DNSimple App to Okta
-
-1. Log in to your organization's Okta marketplace.
-1. Search for DNSimple, and click Add.
-1. Enter an Application Label in General Settings.
-1. Click Done.
-1. Assign yourself to the application.
-1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need them in the next step.
-
-### Configuring DNSimple for Okta SSO
+### Adding the DNSimple App to Okta {#adding-the-dnsimple-app-to-okta}
+
+DNSimple's Okta app integration is pending review and hence is not yet available for installation via the [Okta Integration Network](https://www.okta.com/integrations/). For now, you can already use Okta as an Identity Provider by [creating your own custom Okta app integration](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm) to work with DNSimple, by following the below steps.
+
+1. Log in to your organization's Okta dashboard as an administrator.
+1. Go to the <label>Applications</label> page, then click Create App Integration.
+1. Select <label>OIDC - OpenID Connect</label> as the Sign-in method and <label>Web application</label> as the Application type.
+![Create App Integration](/files/okta-create-app-integration.png)
+1. Enter "DNSimple App Integration" or something appropriate for the <label>App integration name</label> field.
+1. Under <label>Grant type</label>, uncheck "Client Credentials" and keep "Authorization Code" as the only option checked.
+1. Under <label>Sign-in redirect URIs</label>, add the URIs "https://dnsimple.com/identity_providers/okta/callbacks/users/login", "https://dnsimple.com/identity_providers/okta/callbacks/accounts/link" and "https://dnsimple.com/identity_providers/okta/callbacks/users/link"
```

Comment:
This might work better as a list.

```suggestion
1. Under <label>Sign-in redirect URIs</label>, add the URIs:
- https://dnsimple.com/identity_providers/okta/callbacks/users/login
- https://dnsimple.com/identity_providers/okta/callbacks/accounts/link
- https://dnsimple.com/identity_providers/okta/callbacks/users/link
```

Reply:
Great idea! Formatted the URIs as a list here: https://github.com/dnsimple/dnsimple-support/pull/1005/commits/8c56be01aedd0559cd56e8f1950ce00364d87344